Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp

```diff
@@ -128,7 +128,6 @@ void Connection::platformInitialize(Identifier&& identifier)
 void Connection::platformInvalidate()
 {
 #if USE(GLIB)
-    // In the GLib platform the socket descriptor is owned by GSocket.
```

Comment:
This is still true, no? we are still passing the ownership to the GSocket

Reply:
Yes it's still true, but it's no longer _conditionally_ true, so the comment to explain the previously-confusing behavior is no longer useful. Notably `m_socketDescriptor` doesn't exist anymore.

This change really should have landed via https://github.com/WebKit/WebKit/pull/35314 but seems I missed it.